lib/lotus/entity/dirty_tracking.rb

```diff
@@ -25,46 +25,19 @@ module DirtyTracking
       #   article.changed? # => true
       #
       #   article.changed_attributes # => {:title => "Generation P"}
-      def self.included(base)
-        base.class_eval do
-          extend ClassMethods
-        end
-      end
-
-      module ClassMethods
-        # Override attribute accessors function.
-        # Create setter methods with attribute values checking.
-        # If the new value or a changed, added to @changed_attributes.
-        #
-        # @params attr [Symbol] an attribute name
-        #
-        # @since 0.3.1
-        # @api private
-        #
-        # @see Lotus::Entity::ClassMethods#define_attr_accessor
-        def define_attr_accessor(attr)
-          attr_reader(attr)
 
-          class_eval %{
-            def #{ attr }=(value)
-              _attribute_changed(:#{ attr }, @#{ attr }, value)
-              @#{ attr } = value
-            end
-          }
-        end
-      end
+      attr_reader :init_state
```

Comment:
That how I'm protect initial state (@init_state) of entity from outside modifications. But now I thinking that it can conflict with users methods and better hide it state somehow. 
Or we can put it in public API to show init state (then better to rename it to @init_attributes). 
Also it's possible to implement #revert! that can restore entity to initial state in separate PR (not sure it should be).
What do you think?